src/proxy-installers/shadowsocks-installer.ts

```
@@ -34,6 +34,10 @@ export class ShadowsocksInstaller {
               return F(tags[i].replace('shadowbox-', 'ss://'));
             }
           }
+        }).catch((e: Error) => {
+          console.error('Error reading tags: ', e);
+          clearInterval(intervalId);
+          R(e);
```

Shouldn't we propagate the error instead? How are you notifying the user?